raft/log_entry.hpp

```diff
@@ -0,0 +1,58 @@
+// Copyright (C) 2018 Bluzelle
```

Comment:
What happens when there is an error loading our state?

Reply:
I have added some simple checks to the RAFT and log_entry rehydration.
Runtime exceptions with be thrown if
- the JSON messages in log entries cannot be parsed
- state values are not loaded
- after rehydrating the log entries, the state values do not agree with the values loaded from the entries log.